esm/b64.js

```diff
@@ -0,0 +1,41 @@
+var U = Uint8Array,
```

Comment:
This whole file looks copy/pasted from a minified source + I am not even sure this is necessary or any better than `atob` and `btoa`.

I won't accept this code as is so please consider:
  * use [my own code](https://github.com/WebReflection/uint8-to-base64#readme) instead
  * provide full source code of this hand crafted minified code
  * use `atob` and `btoa` or anything natively available with Node.js fallback such as
```js
const {
  btoa = str => Buffer.from(str).toString('base64'),
  atob = str => Buffer.from(str, 'base64').toString()
} = globalThis;
```

Reply:
Well, 
- i would not like to want to use `node:buffer` cuz that would increase the web bundle too much with junk and then it would not be so cross env friendly for either Deno or Browsers

- atob/btoa are pretty terrable to use (and slow). in fact i bench tested your own module with the one in this PR and yours where a magnitude slower...

```js
// import { fromByteArray, toByteArray } from 'https://jspm.dev/base64-js'
// import fastBase64decode from 'https://jspm.dev/fast-base64-decode'
// import { encode as base64ArrayBufferEncode } from 'https://jspm.dev/base64-arraybuffer'
import * as uint from 'https://jspm.dev/uint8-to-base64'
import { encode, decode } from './base64.js'

const randomNumber = crypto.getRandomValues(new Uint16Array(1))[0]
console.log(`creating ${randomNumber} random bytes`)
const randomBytes = crypto.getRandomValues(new Uint8Array(randomNumber))

const b64 = encode(randomBytes)
const b64u = uint.encode(randomBytes)

Deno.bench('current b64 encode', () => {
  let i = 25
  while(i--) encode(randomBytes)
})

Deno.bench('your minimal b64 encode', () => {
  let i = 25
  while(i--) uint.encode(randomBytes)
})

Deno.bench('current b64 decode', () => {
  let i = 25
  while(i--) decode(b64)
})

Deno.bench('your minimal b64 decode', () => {
  let i = 25
  while(i--) uint.decode(b64u)
})
```

creating 15952 random bytes

bench current b64 encode ... 1000 iterations 798,574 ns/iter (703,917..920,333 ns/iter) ok **(1s)**
bench your minimal b64 encode ... 1000 iterations 5,960,122 ns/iter (5,615,625..6,276,500 ns/iter) ok **(11s)**

bench current b64 decode ... 1000 iterations 891,337 ns/iter (819,000..1,106,417 ns/iter) ok **(1s)**
bench your minimal b64 decode ... 1000 iterations 19,531,555 ns/iter (18,868,916..21,563,958 ns/iter) ok **(39s)**

bench result: ok. 4 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out **(54s)**

Reply:
I no longer have the original source and don't really remember where i got it from. This is something that i had laying around as being one of the fastest base64 encoder/decoder in user land that i had previously been used and run closure compiler on do further optimize the speed. but i can make it look not so minified if you like.

Reply:
> I no longer have the original source and don't really remember where i got it from

this is a blocker for this project, sorry. Please use native/primitives even if not as fast as user-land code, which is pretty silly to me to hear that ... *bun* has native btoa and atob implementation, as example, while Node.js explicitly states *no optimizations because we don't care* on their side ... 

Last, but not least, apparently node.js has native atob and btoa since 16 LTS so you don't need the fallback I've mentioned before.

As summary: I won't use unknown code potentially stolen from the Web without credits in this, or any of my, projects.

Reply:
P.S. the following

```js
const {
  btoa = str => Buffer.from(str).toString('base64'),
  atob = str => Buffer.from(str, 'base64').toString()
} = globalThis;
```

requires zero bloat on browsers because browsers won't ever reach those fallbacks ... your bundling should know this too if you set it as browser target ;-)